src/test/java/ibm/jceplus/junit/openjceplus/TestECDHKeyAgreementParamValidation.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright IBM Corp. 2024
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+package ibm.jceplus.junit.openjceplus;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public class TestECDHKeyAgreementParamValidation extends ibm.jceplus.junit.base.BaseTestECDHKeyAgreementParamValidation{
```

Comment:
I think it would be better if we import the base class and use the simple name here.

Reply:
updated